FEXCore/Source/Interface/Core/CPUBackend.h

```diff
@@ -144,31 +177,37 @@ namespace CPU {
 
     bool IsAddressInCodeBuffer(uintptr_t Address) const;
 
+    fextl::shared_ptr<CodeBuffer> GetThreadLocalCodeBuffer() const;
```

Comment:
With other comments I think this could be dropped entirely (and in ctor it can populate the initial shared member)